cookiecutter-django-app/{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -55,8 +56,8 @@ commands =
     doc8 --ignore-path docs/_build README.rst docs
     rm -f docs/{{ cookiecutter.app_name }}.rst
     rm -f docs/modules.rst
-    make -C docs clean
-    make -C docs html
+    make -e -C docs clean
+    make -e -C docs html
```

Comment:
Any reason not to just put `SPHINXOPTS` into the Makefile?

Reply:
I was wondering that also.

Reply:
The Makefiles were initially generated so I chose to leave them alone and opted for putting the overrides in our tox file.

Reply:
Ah, I see. Probably worth a comment here.